nucliadb_node/src/services/reader.rs

```diff
@@ -93,11 +95,14 @@ impl ShardReaderService {
     }
     #[tracing::instrument(skip_all)]
     pub fn get_info(&self, request: &GetShardRequest) -> NodeResult<Shard> {
+        let span = tracing::Span::current();
```

Comment:
If time we measure for `request_time` is always in the scope of a function, we might want to create a simple macro to decorate it and automatically handle time and metric. Similar to the observer decorator in the Python side